lib/readwise/client.rb

```diff
@@ -21,6 +23,14 @@ def create_highlights(highlights: [])
       raise NotImplementedError
     end
 
+    def get_highlight(highlight_id:)
```

Comment:
Can we write a test for this new method?

Example: https://github.com/andjosh/readwise-ruby/blob/054f040acccdce26ef52d9fbd250995918b1b313/spec/readwise/client_spec.rb#L13-L22

Reply:
Absolutely, cranking on this now!